dhall/ghc-src/Dhall/Crypto.hs

```diff
@@ -8,13 +9,13 @@ module Dhall.Crypto (
     ) where
 
 import Control.DeepSeq (NFData)
-import Crypto.Hash (SHA256)
+import "cryptonite" Crypto.Hash (SHA256) -- To avoid conflict with "hashing"
```

Comment:
That's a bit unclear – we don't directly depend on `hashing` after all.

Reply:
Is the use of `PackageImports` still necessary now that the `stack` issue was resolved?  My understanding is that this disambiguation should not be necessary

Reply:
I realized I may not have the most recent version `stack`. I'll check when I get home.